make/release.mk

```
@@ -4,6 +4,7 @@ REPO_ARCHIVE_FILE := $(BUILD_DIR)/dkp-insights.tar.gz
 CATALOG_IMAGES_TXT := $(BUILD_DIR)/dkp_insights_images.txt
 INSIGHTS_CATALOG_APPLICATIONS_CHART_BUNDLE := $(BUILD_DIR)/dkp-insights-charts-bundle.tar.gz
 RELEASE_S3_BUCKET ?= downloads.mesosphere.io
+REPO_ROOT := $(CURDIR)
```

(nit) It might be safer to use this:
```suggestion
REPO_ROOT := $(shell git rev-parse --show-toplevel)
```

---

For some reason, when I'm using he suggested `git config` command, builds still fail with `fatal: detected dubious ownership in repository at` error. (example [build 1](https://github.com/mesosphere/insights-catalog-applications/actions/runs/4202601050/jobs/7290957564), example [build 2](https://github.com/mesosphere/insights-catalog-applications/actions/runs/4202601050/jobs/7290957679))
The actual value of the ROOT_DIR points to the same variable in both cases, though.
Builds pass with no issues when using `CURDIR` instead

---

Okay, we can stick to `CURDIR`